File: social-service/src/test/java/com/eirsteir/coffeewithme/config/SocialServiceTestConfiguration.java
package com.eirsteir.coffeewithme.config;


import com.eirsteir.coffeewithme.social.config.SocialServiceConfiguration;
import io.eventuate.tram.spring.consumer.common.TramConsumerCommonConfiguration;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

@Configuration
@EnableAutoConfiguration
@Import({SocialServiceConfiguration.class,
        TramConsumerCommonConfiguration.class})
public class SocialServiceTestConfiguration {
}

File: social-service/src/test/kotlin/com/eirsteir/coffeewithme/config/SocialServiceTestConfiguration.kt
package com.eirsteir.coffeewithme.config

import com.eirsteir.coffeewithme.social.config.SocialServiceConfiguration
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import

@Configuration
@EnableAutoConfiguration
@Import(
    SocialServiceConfiguration::class, TramConsumerCommonConfiguration::class
)
class SocialServiceTestConfiguration 